main.py

```diff
@@ -140,8 +150,8 @@ def check_signed_up(self, email):           # Returns false if there is already
     def get_schedule_for_name(self, firstname, lastname):
         schedule_data = get_schedule_data()
         for schedule in schedule_data:
-            if schedule['firstname'].lower() == firstname.lower() and \
-               schedule['lastname'].lower() == lastname.lower(): # If the schedule is the requested schedule
+            if normalize_name(schedule['firstname']) == firstname.lower() and \
```

Comment:
Some sort of normalize_compare(x, y) function would be even better, but we can consider that later.